Experiments_toolkit.py

```diff
@@ -57,151 +50,143 @@ def splitting_up_data(dictionary, dict_key, train, percent_train, test, percent_
-                                                        finding the precise start and end time of a lift in the data)
-            window_size -- int (Window size of the trace to be saved into the dictionary that will be
-                                                returned)
-            offset -- int (Number to add to the predicted start time of the lift from where the window
-                                                will start at)
+                                stores important parameters for the 2 steps involved with 
+                                finding the precise start and end time of a activity in the data)
+            window_size -- int (Window size of the trace to be saved into the
+            				    dictionary that will be returned)
+            offset -- int (Number to add to the predicted start time of the 
+            			   activity from where the window will start at)
 
        Return:
-           windowed_data_dictionary -- dict (Dictionary that stores the windows under the "windows" key)
+           windows-- int[] (Array of the windows cropped by the window_function)
            """
-    windowed_data_dictionary = {"windows": []}
-    for j in range(len(DATA)):
-        sample = DATA[j]
-        lift_windows = wl.initial_find_lift(
-            sample, rows, filtering_params.initial_filtering.window_size, filtering_params.initial_filtering.stride, filtering_params.initial_filtering.divisor)
-        if len(lift_windows) > 0:
+    windows = []
+    for j in range(len(data)):
+        sample = data[j]
+        activity_windows = wl.initial_find_activity(
+            sample, rows, filtering_params.initial_filtering.window_size,
+            filtering_params.initial_filtering.stride,
+            filtering_params.initial_filtering.divisor)
+        if len(activity_windows) > 0:
             start_time, end_time = wl.get_start_time(
-                lift_windows, rows, filtering_params.precise_filtering.window_size, filtering_params.precise_filtering.stride, filtering_params.precise_filtering.threshold)
+                activity_windows, rows, filtering_params.precise_filtering.window_size,
+                filtering_params.precise_filtering.stride,
+                filtering_params.precise_filtering.threshold)
             pos_window = window_function(
                 sample, start_time + offset, window_size)
             if len(pos_window[0][0]) != window_size:
                 continue
-            windowed_data_dictionary["windows"].extend(pos_window)
-    return windowed_data_dictionary
+            windows.extend(pos_window)
+    np_windows = np.array(windows)
+    return np_windows
 
 
 def get_np_X_Y(x, y, length):
     """
-       Description: Create X and Y numpy arrays desired length of random samples from lists of windowed data, x and y
+       Description: Create X and Y numpy arrays desired length of random 
```

Comment:
I think you're sampling without replacement? Would be good to make that explicit, since it will affect the max number of samples that can be drawn